aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputFile.java

```diff
@@ -57,7 +57,7 @@ public PositionOutputStream createOrOverwrite() {
     try {
       return new S3OutputStream(client(), uri(), awsProperties());
     } catch (IOException e) {
-      throw new UncheckedIOException("Filed to create output stream for location: " + uri(), e);
```

Comment:
thanks for noticing this! But I think we should not have it in this PR.

Reply:
Yes,  we could just address the s3 typo in a separate PR.  Let's just revert it here.